### PR TITLE
120 i18n component change mycohortdetails

### PR DESCRIPTION
**Pull request by PeachyOmnivore**
https://trello.com/c/scZez88R/124-i18n-component-changes

### CODE REVIEW

File: src/components/header/index.js
Comment (Uprising5034) [inline src/components/header/index.js:1]: wrong file comitted here?

File: src/locales/en/translation.json
Comment (Uprising5034) [inline src/locales/en/translation.json:1]: These translation files also appear to have the same commits as from the header issue.

**Review by Uprising5034 — CHANGES_REQUESTED** [review]

**Comment by Uprising5034** [comment]
Also missing swedish translation file.

File: src/locales/en/translation.json
Comment (PeachyOmnivore) [reply src/locales/en/translation.json:1]: all linked up.

**Pull request merged**